pages/common/git-show.md

```diff
@@ -19,10 +19,14 @@
 
 `git show {{branch}}~{{3}}`
 
-- Show a commit's hash and message in a single line, suppressing the diff output:
+- Show a commit's message in a single line, suppressing the diff output:
```

Comment:
The short hash is shown in the output, so why change this?
```
git show --oneline -s bf3503d258865b5344128e8ffccb34435a3e8b6c
bf3503d Fixes
```

Reply:
As I mentioned in the PR description:

> to avoid misleading people regarding when the commit hash is shown.

The hash was added to the first example to make it clearer that it's shown by default, rather than specifically in this example.

Reply:
Ok :)